package ru.itmo.tgbot.exception.event

class EventAlreadyExistsException(cause: Throwable): RuntimeException(cause)